### ISSUE TITLE
Turn off logger

### ISSUE THREAD

**Issue by anthonator** [gen-smtp/gen_smtp#199]
I'd like to prevent the logging that occurs on `:gen_smtp_server` start. I'm assuming it's ranch that's responsible for those messages but setting the `:ranch_opts` `:logger` option to `false`, `nil` or a random atom doesn't have any effect.

**Comment by seriyps** [comment]
What are the log messages you want to disable?

**Comment by anthonator** [comment]
```
13:25:37.113 [info]  :gen_smtp_server starting at :nonode@nohost

13:25:37.116 [info]  :gen_smtp_server listening on {127, 0, 0, 1}:53652 via :tcp
```

**Comment by seriyps** [comment]
You are using older version of gen_smtp

https://github.com/gen-smtp/gen_smtp/blob/813d0258eeac6b9cc4ce47b7a9616ec5f8e43c80/src/gen_smtp_server.erl#L151

Ranch was introduced relatively recently

**Comment by anthonator** [comment]
I see. I'm assuming the newer version of `gen_smtp` is on `master` and not Hex.pm?

Is there any way to turn off these messages on older versions of `gen_smtp`?

**Comment by seriyps** [comment]
Looks like there is still 1 issue in milestone for it to be published.
https://github.com/gen-smtp/gen_smtp/milestone/2

I don't know how to disable those logs unfortunately.

**Comment by anthonator** [comment]
Thanks for the feedback!

My use case is geared towards launching a `gen_smtp_server` for test purposes in an Elixir application and my output is getting mucked up with these log messages. Would love to figure something out. Even for older versions.

**Comment by seriyps** [comment]
those messages are generated using deprecated OTP `error_logger` API. In newer OTP versions `error_logger` calls are redirected to new OTP `logger` framework. I don't have experience with it yet, but I think this page can help http://erlang.org/doc/apps/kernel/logger_chapter.html#backwards-compatibility-with-error_logger .

Also, I don't know how Erlang's logging system is interacting with Elixir, maybe there are some other ways

**Comment by mworrell** [comment]
I will have a look at the remaining issue.

**Comment by mworrell** [comment]
I have rebased and updated the remaining merge request #184 
If that one is merged then we can release a 1.0.

**Comment by mworrell** [comment]
Closing this issue for now - as I don't see anything that we should change at the moment?
